test/Microsoft.Sbom.Parsers.Spdx22SbomParser.Tests/Parser/SbomFileParserTests.cs

```diff
@@ -100,7 +100,6 @@ public void MissingPropertiesTest_ThrowsSHA256()
         Assert.ThrowsException<ParserException>(() => result.Files.Select(f => f.ToSbomFile()).ToList());
     }
 
-    [DataTestMethod]
```

Comment:
For consistency, can you please put the `[TestMethod]` attribute _before_ the `DataRow` attributes for the tests in this file?